react-accordion-component/my-app/src/Accordion.js

```diff
@@ -0,0 +1,22 @@
+import Topics from './Topics';
+import { useState } from 'react';
+
+export default function Accordion({ topics }) {
+  const [clicked, setClicked] = useState('');
```

Comment:
`clicked` is ambiguous. You are really keeping track of which item should be showing, so `openItem` or `showing` or similar would be better.